### PR TITLE
Add a stack.yaml file and use LTS-9.0

### LINKED ISSUE #8: Bump upper version of base
The package needs to bump upper version of `base` and it would be good to add it to Stackage.

### PR DESCRIPTION
**Pull request by goetzc**
Also bumped the version to 0.1.1.1, not sure if it's the right one. Closes #8

### CODE REVIEW

File: dbcleaner.cabal [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name:           dbcleaner
-version:        0.1.1
+version:        0.1.1.1
```
Comment (sestrella) [inline dbcleaner.cabal]: @goetzc `0.1.2` instead?

Comment (goetzc) [reply dbcleaner.cabal]: Sure

**Review by sestrella — APPROVED** [review]
@goetzc LGTM

**Comment by sestrella** [comment]
@goetzc could you please add this package to https://github.com/fpco/stackage/blob/master/build-constraints.yaml#L1899 after uploading the new version to Hackage.

**Comment by goetzc** [comment]
([PR](https://github.com/fpco/stackage/pull/2685) to add it to stackage.)

**Pull request merged**